skynet/bandwidth_test.go

```diff
@@ -24,18 +33,40 @@ func TestNumChunks(t *testing.T) {
 	}
 }
 
+// TestRawStorageUsed ensures that RawStorageUsed works as expected.
+func TestRawStorageUsed(t *testing.T) {
+	tests := []struct {
+		size   int64
+		result int64
+	}{
+		{size: 0, result: baseSectorTotalSize},
+		{size: 1 * MiB, result: baseSectorTotalSize},
+		{size: 4 * MiB, result: baseSectorTotalSize},
+		{size: 5 * MiB, result: baseSectorTotalSize + chunkTotalSize},
+		{size: 50 * MiB, result: baseSectorTotalSize + 2*chunkTotalSize},
+		{size: 500 * MiB, result: baseSectorTotalSize + 13*chunkTotalSize},
```

Comment:
There should be a comment explaining why this is 13

Reply:
Added comments.